### ISSUE TITLE
a question about the spectrum of sph_est.wav

### ISSUE THREAD

**Issue by Captain2xxx-coder** [JupiterEthan/GCRN-complex#3]
Hi, 

it's a great job and helps me a lot !
But I have come across a question that the spectrum of estimated speech (sph_est.wav) has stripes shown in the following picture.
![image](https://user-images.githubusercontent.com/79581235/171319318-67a4b47a-3a51-4a8c-96a9-e46622663147.png)
Besides, my validation set loss didn't change during training and was different from the training set loss. Do you have any ideas to figure out these problems?

Thank you so much!

**Comment by JupiterEthan** [comment]
Hello, the stripes look like checkerboard artifacts caused by strided convolution. Have you tried to set the kernel size to a multiple of the stride size for all convolution and deconvolution? For example, if you use a stride of (1, 2) for convolution/deconvolution, you can set the kernel size to (1, 4) instead of (1, 3). This will likely alleviate the problem.

Regarding the validaiton loss issue, did you apply any normalization/rescaling to the data? If so, did you make sure that the way of scaling/normalization is the same between training and validation?

**Comment by QQY9** [comment]
> Hi, 你好，
> 
> it's a great job and helps me a lot !这是一份很棒的工作，对我帮助很大！ But I have come across a question that the spectrum of estimated speech (sph_est.wav) has stripes shown in the following picture. 但我遇到一个问题，估计语音的频谱（sph_est.wav）有如下图所示的条纹。 ![image](https://user-images.githubusercontent.com/79581235/171319318-67a4b47a-3a51-4a8c-96a9-e46622663147.png) Besides, my validation set loss didn't change during training and was different from the training set loss. Do you have any ideas to figure out these problems? 此外，我的验证集损失在训练期间没有改变，并且与训练集损失不同。您有什么想法来解决这些问题吗？
> 
> Thank you so much!太感谢了！

Hi, is the problem solved that the validation set loss did not change during the training period
